saleor/graphql/account/tests/test_account.py

```diff
@@ -4444,8 +4444,8 @@ def test_password_change_invalid_new_password(user_api_client, settings):
 
 
 ADDRESS_CREATE_MUTATION = """
-    mutation CreateUserAddress($user: ID!, $city: String!, $country: CountryCode!) {
-        addressCreate(userId: $user, input: {city: $city, country: $country}) {
+    mutation CreateUserAddress($user: ID!, $address: AddressInput!) {
```

Comment:
Can you add a test that will raise an exception in case of providing incorrect address?

Reply:
added .